.travis.yml

```diff
@@ -1,12 +1,8 @@
 language: php
 
 php:
-  - 5.3
```

Comment:
IMO this would be the same like
```
     - php: 7.0
       env: SYMFONY_VERSION=3.0.*
```
The latest symfony version is 3.0 @Soullivaneuh